base/common/python/pki/nssdb.py

```diff
@@ -2009,6 +2009,8 @@ def get_cert_info(self, nickname, token=None):
         cert = {}
         cert['object'] = cert_obj
 
+        cert['data'] = self.get_cert(nickname=nickname, token=token, output_format='base64')
```

Comment:
We probably can reuse the return value from `get_cert()` in line 2001, then convert it to base64 with `convert_cert()`.

Reply:
I have tried to use the `cert_pem` value but it is a binary string and the `convert_cert()` has some trouble to convert.

Reply:
OK, no worries then, we can deal with it another time. Thanks!